nmkapp/templates/profile.html

```diff
@@ -139,9 +139,20 @@ <h2 class="content-header">{% trans "E-mail notifications" %}</h2>
 </form>
 <br/>
 
-<h1 class="content-header">{% trans "CHANGE PASSWORD" %}</h1>
+{% if current_user.has_usable_password %}
```

Comment:
i ljudi koji sutra dodju preko FB-a i sl. ce isto moci da odrade "create password"? to mi je ok, ne treba da se blokiras ovde, nego samo da razumem gde su sve izmene ako dodajemo novi auth

Reply:
Trebalo bi da bilo koji third-party user moze da pravi sifru. Jer sumnjam da ce bilo koji servis da nam da njihovu sifru(hash). Tako da ce to uvek da bude True za njih